connect_four/board.rb

```diff
@@ -0,0 +1,151 @@
+class Board
+  def initialize
+    @board_width = 7
+    @board_height = 6
+    @board = Array.new(@board_width){Array.new(@board_height)}
+  end
+
+  def drop_token(column_number,game_token)
+    board_full

```

Comment:
It looks like a Boolean method? Add a question mark? And pronounce it with upspeak? Okay?